### DISCUSSION TITLE
QAnything 2.0.0 如何部署自定义模型如qwen-7b，有没有详细的教程呢

### DISCUSSION THREAD

**Discussion by poutestcase** [netease-youdao/QAnything#509] [Q&A]
QAnything 2.0.0 如何部署自定义模型如qwen-7b，有没有详细的教程呢

**Comment by TatyOko28** [comment]
To deploy a custom model like Qwen-7B in QAnything 2.0.0, you need to follow these steps:

Install Required Dependencies for QAnything and the Qwen model.
Prepare the Qwen Model (e.g., downloading weights, tokenizers, etc.).
Integrate Qwen-7B into QAnything's Pipeline.
Configure and Deploy the model using QAnything's configuration.
